vmm/src/config.rs

```diff
@@ -72,6 +73,52 @@ pub enum Error {
     /// Failed to parse vsock parameters
     ParseVsock(OptionParserError),
 }
+
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        use self::Error::*;
+        match self {
+            ParseConsole(o) => write!(f, "Error parsing --console: {}", o),
+            ParseConsoleFileMissing => {
+                write!(f, "Error parsing --console: path missing when using file")
+            }
+            ParseConsoleInvalidModeGiven => {
+                write!(f, "Error parsing --console: invalid console mode given")
+            }
+            ParseCpus(o) => write!(f, "Error parsing --cpus: {}", o),
+            ParseCpusMaxLowerThanBoot => {
+                write!(f, "Error parsing --cpus: max CPUs greater than boot CPUs")
+            }
+            ParseDevice(o) => write!(f, "Error parsing --device: {}", o),
+            ParseDevicePathMissing => write!(f, "Error parsing --device: path missing"),
+            ParseDiskSocketAndPath => write!(
+                f,
+                "Error parsing --disk: vhost socket and disk path both provided"
+            ),
+            ParseFileSystem(o) => write!(f, "Error parsing --fs: {}", o),
+            ParseFsSockMissing => write!(f, "Error parsing --fs: sock missing"),
+            ParseFsTagMissing => write!(f, "Error parsing --fs: tag missing"),
```

Comment:
You could factorize every `Error parsing --fs/net/console...` to avoid repeating the same string every time.

Reply:
I have an idea but I would like to do it on a follow-up PR.

Reply:
Fine by me!